storage/redis_cluster.go

```diff
@@ -1298,3 +1298,67 @@ func (r *RedisCluster) RemoveSortedSetRange(keyName, scoreFrom, scoreTo string)
 func (r *RedisCluster) ControllerInitiated() bool {
 	return r.RedisController != nil
 }
+
+// ScanKeys will return all keys according to the pattern.
+func (r *RedisCluster) ScanKeys(pattern string) ([]string, error) {
```

Comment:
Why don't we use Universal client here and built-in redis library function?

Reply:
This is a wrapper so that the operation based on different cluster config is handled here. 
Directly calling redis universal client means we have to repeat later.

Reply:
@furkansenharputlu I've also found an improvement, refactored `GetKeys` to use `Scan` method internally, `GetKeys` essentially do the same thing except some minor things for keys.